docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,21 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## Data Ingestion Metrics
+
+| **Category** | **Definition** | **KPI** |
+|----------|-----------|-----|
+| Data Completeness | Measures the extent to which all required data fields are present in the dataset and tracks the overall data completeness trend over a specified period.| Percentage of data fields that are complete and the overall data completeness over time. |
+| Data Quality Performance | Measures the percentage of successful data quality checks and the total number of data quality checks conducted within a specified period. | Number of data quality checks conducted and the percentage of successful data quality checks . |
+|Data Ingestion Frequency | Measures how often data is ingested into the system. | Number of data ingestion events per unit of time (daily, weekly, monthly, quarterly, annually). |
+| Volume of Data Ingested | Measures the total volume of data ingested into the repository. | Total volume of data ingested into the repository.  |
+| Growth Rate | Measure the rate at which the volume of data ingested is increasing over time. | Percentage increase of total data volume in repository per unit of time. |
+| Ingestion Latency | Measures the average time taken for data to be ingested into the repository and tracks the trend of this latency over a specified period. | Mean time of data ingestion latency and the latency trend over a specified period. |
```

Comment:
I like this one. A few thoughts:
1. Do we need to call out that latency may differ by dataset? 
2. Do you intend to use "mean" time? Not average or percentile? All have merits, so just confirming.
3. This can likely be split into multiple KPIs.
4. Is latency trend a KPI or a visualization of a KPI over time? Not sure if visualizations need to be called out here unless we need to speak to the value of the visual. I'm open to either approach. Just thinking out loud to keep this simple. If we do keep it, it's probably a separate KPI that might be better if we can quantify a single number for it. Not sure 🤔

Reply:
I have added per dataset in the Formula column, adjusted to average instead of mean time, and have removed the trend component as this is a result of the KPI